src/olympia/addons/serializers.py

```diff
@@ -162,13 +162,12 @@ class SimpleVersionSerializer(MinimalVersionSerializer):
     is_strict_compatibility_enabled = serializers.SerializerMethodField()
     license = CompactLicenseSerializer()
     release_notes = TranslationSerializerField()
-    url = serializers.SerializerMethodField()
```

Comment:
I was worried about breaking the api but it seems we never documented url in response anyway!

Reply:
Actually, I make use of it in https://github.com/mozilla/addons-server/pull/10684/files#diff-461691f0e19e8fe3e4d86b4b69858cb5R208

Reply:
So, maybe, if we can keep it? Otherwise I'll re-implement it for my own serializer there.

Reply:
On the other hand… I'll just drop it from my PR too, I don't think we'll need it anyway.

Reply:
yeah I dropped it because there isn't a (publicly accessible) unique url for a version any more.  Just returning the /versions page for each version seemed redundant.